### ISSUE TITLE
Provide User Help

### ISSUE THREAD

**Issue by ianhewubs** [paranext/paranext-core#1155]
## USER STORY
As a user I need some guidance in Paratext 10 for how to use features in PT10. This will be true when the product is mature. It is especially true when the product is new and there is no support or documentation.

- Decide whether this is offline only, or offline and online
- Off the shelf or in house developed
- Choose how to deliver Help in the UI - selecting from possibilities such as tooltips, guides, help topics, links to support.bible, etc.
- Research most effective Help delivery in modern software

**Comment by tjcouch-sil** [comment]
I propose we consider creating a system in which people can define tutorial scripts that the software can walk the user through. The tutorials could show something in a modal, dim the screen and highlight just one thing to click on with a description box near it, etc. Thinking this might be effective since the software is so modular that everyone's UI will probably look different. Instead of showing a video where the UI looks some way because it has extensions X, Y, and Z, we just load up the tutorial right in the user's software, and the menu items will be wherever they actually are for that user. Just one idea :) I imagine this would probably take significant effort.

**Comment by ianhewubs** [comment]
@tjcouch-sil Yes, "tour-guide" helps are an option I'd like to explore, especially since there are off the shelf options for these. I suspect we will ultimately need to provide Help in more than one format. Perhaps a combination of low and higher effort solutions.
